### PR TITLE
Use Roboto Font

### PR DESCRIPTION
**Pull request by fiennyangeln**
Use roboto font using Fout Stager and MuiThemeProvider

Bug #35

### CODE REVIEW

File: src/components/FontStager/index.css [hunk 1]
```diff
@@ -0,0 +1,58 @@
+@font-face {
+    font-family: Roboto500;
+    src:
+            url('~typeface-roboto/files/roboto-latin-500.woff2') format('woff2'),
+            url('~typeface-roboto/files/roboto-latin-500.woff') format('woff');
```
Comment (helfi92) [inline src/components/FontStager/index.css]: Fix indentation.

File: src/components/FontStager/index.css [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    font-family: Roboto500, sans-serif;
+}
+
+/* Prevent the secondary fonts from being tree-shaken away */
+.font-stage-secondary .roboto300 {
+    font-family: Roboto300, sans-serif;
+}
+.font-stage-secondary .roboto700 {
+    font-family: Roboto700, sans-serif;
+}
```
Comment (helfi92) [inline src/components/FontStager/index.css]: According to the [docs](https://material-ui.com/style/typography/#roboto-font-cdn), Material-UI default typography configuration only relies on 300, 400 and 500 font weights. This file is using weight 700. I think we can just use https://github.com/taskcluster/taskcluster-web/blob/a7cc659411351296a42243fa835ae9ae9164c99b/src/components/FontStager/index.css.

**Comment by fiennyangeln** [comment]
@helfi92 @eliperelman ok! :D

File: src/App.jsx [hunk 3]
```diff
@@ -28,14 +31,17 @@ export default class App extends Component {
...
-        <BrowserRouter>
-          <Switch>
-            <Route path="/projects/:project" component={Project} />
-            <Route exact path="/" component={Projects} />
-          </Switch>
-        </BrowserRouter>
-      </ApolloProvider>
+      <MuiThemeProvider theme={theme}>
+        <FontStager />
+        <ApolloProvider client={this.apolloClient}>
```
Comment (helfi92) [inline src/App.jsx]: Let's make `ApolloProvider` the root component.

```jsx
<ApolloProvider ...?
  <MuiThemeProvider ... >
    <FontStager ... >
    <BrowserRouter>
```

**Pull request merged**